include/dxc/dxcapi.h

```diff
@@ -306,7 +415,9 @@ struct IDxcCompiler : public IUnknown {
     _COM_Outptr_ IDxcOperationResult **ppResult   // Compiler output status, buffer, and errors
   ) = 0;
 
-  // Preprocess source text
+  /// \brief Preprocess source text
+  ///
+  /// \todo Is IDxcCompiler::Preprocess() deprecated too?
```

Comment:
Note: all these 'deprecated' interfaces will still work (implemented as a wrapper on the IDxcCompiler3 implementation).  They are the old interfaces though, so using IDxcCompiler3 is recommended going forward (to avoid translation layers).

For `IDxcCompiler::Preprocess()` specifically, the new interface supports this by using the `Compile()` method with the `-P` argument.

Reply:
Thanks; goal for deprecated tag here is to point people towards the interface/method that we want them to be using instead and to not spend time documenting the old ones.